conf/i18n/de/sunrise.yaml

```diff
@@ -1,3 +1,5 @@
 payment:
   prepaid: Vorkasse
-  creditcard: Kreditkarte
\ No newline at end of file
+  creditcard: Kreditkarte
+changePassword:
+    messageSentTitle: Eine Wiederherstellungs-E-Mail wurde gesendet, bitte überprüfen Sie Ihre E-Mail für Anweisungen zum Zurücksetzen Ihres Passworts.
```

Comment:
Sorry I missed it last time, but this message should go to my-account.yaml (there's already a section for `changePassword`).
sunrise.yaml is a file that doesn't go to the Sunrise Theme, therefore it gets lost here. It's fine for the payment because it's very likely going to be removed to something more functional ;)

Reply:
Sounds good to me 👍